packages/esm-service-queues-app/src/config-schema.ts

```diff
@@ -122,13 +122,9 @@ export const configSchema = {
   },
   customPatientChartUrl: {
     _type: Type.String,
-    _default: '',
-    _description: 'Custom URL to load patient chart',
-  },
-  customPatientIdUrl: {
-    _type: Type.String,
-    _default: '',
-    _description: 'Custom URL to fetch patient with id',
+    _default: '${openmrsSpaBase}/patient/${patientUuid}/chart',
+    _description:
+      'Template URL that will be used when clicking on the patient name in the queues table. Available argument: patientUuid',
```

Comment:
```suggestion
      'Template URL that will be used when clicking on the patient name in the queues table. Available argument: patientUuid',
    _validators: [validators.isUrlWithTemplateParameters(['patientUuid']),
```

You'll need to add the import line. Shame that there's no validators used in this config schema yet but it's never too late to start :)

Reply:
Possibly worth noting that both `${openmrsBase}` and `${openmrsSpaBase}` are also available (default provided by the underlying API; but having this front and center is helpful for people trying to customize the URL, e.g., you can do `${openmrsBase}/coreapps/clinicianfacing/patient.page?patientId=${patientUuid}`.

Reply:
This information is front and center by default in the implementer tools... but I understand the implementer tools are at present somewhat orphaned. Though I think they are very relevant to PIH's needs in particular.

Reply:
I'll add those. I didn't know about the `_validators` field. Should we try to type the `configschema` for each app more? I see this here:

https://github.com/openmrs/openmrs-esm-core/blob/b52bba717365134541b4fb7a719f5e5040c4afb7/packages/esm-config/src/types.ts

Reply:
Yeah, use validators when it makes sense to. Please read the [config system documentation](https://o3-docs.openmrs.org/docs/configuration-system).